packages/shared/lib/sdk/response.saver.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import type { ProxyConfiguration } from './sync.js';
+import type { SyncConfig } from '../models/Sync.js';
+
+function ensureDirectoryExists(directoryName: string): void {
+    if (!fs.existsSync(directoryName)) {
+        fs.mkdirSync(directoryName, { recursive: true });
+    }
+}
+
+function getPaginateFilePath(directoryName: string, config: ProxyConfiguration, syncConfig?: SyncConfig): string {
+    const paginateType = config.method?.toLowerCase() || 'get';
+    const fileName = config.endpoint.replace('/', '');
+    const paginatePath = `${directoryName}/mocks/paginate/${paginateType}/${syncConfig?.sync_name}`;
+
+    ensureDirectoryExists(paginatePath);
+
+    if (fileName.includes('/')) {
```

Comment:
how can it contain / if you replaced all of them L14?

Reply:
Didn't replace all but rather the first occurrence of it. So this string: `scheduled_events/7ab23fe1-3047-4a16-9325-93a19e73cb42/invitees` would enter that block